game/world/triggers/abstracttrigger.cpp

```diff
@@ -78,6 +82,7 @@ void AbstractTrigger::processEvent(const TriggerEvent& evt) {
   if(fireDelay>0) {
     TriggerEvent ex(evt.target, evt.emitter, world.tickCount() + fireDelay, evt.type);
     delayedEvent = std::move(ex);
+    world.addDefTrigger(*this);
```

Comment:
why this code with `WorldObjects::addDefTrigger` exists?
tracking if deferred triggers is already implemented outsize of trigger class, that should be enough.

Reply:
Problem is `AbstractTrigger::processEvent` is called directly from `onIntersect` and `onCollide` functions for triggers activated by touch/collision which prevents them being added to `defTriggers`.  Idea was to move adding directly after fireDelay check.